wagtail/wagtailcore/models.py

```diff
@@ -365,6 +365,14 @@ class Page(six.with_metaclass(PageBase, MP_Node, ClusterableModel, index.Indexed
         editable=False
     )
 
+    old_url = models.TextField(
```

Comment:
It gets replaced when moving a page with the new URL so I don't think it's possible to get the old URL this way.